index.js

```diff
@@ -1095,10 +1098,15 @@ Multi.prototype.exec = function (callback) {
                     args[args.length - 1](null, reply);
                 }
             }
+        } else if (err) {
+            err.code = 'EXECABORT';
+            err.errors = errors;
         }
 
         if (callback) {
-            callback(null, replies);
+            callback(err, replies);
```

Comment:
After thinking about it further I'd say it is probably the best to return either err or res. Not both. Therefor if the result is returned it might contain errors as well. But at least they are indeed errors and not strings as they were before.
@bcoe what would you say?